src/Annotation/ApiResource.php

```diff
@@ -62,4 +93,119 @@
      * @var array
      */
     public $attributes = [];
+
+    /**
+     * @var string
+     */
+    private $accessControl;
```

Comment:
Can you add a reference to the PHPStorm plugin issue, so we'll not forget and drop those properties later :)

Reply:
Done.